packages/docs/src/components/landing/SeamlessIntegration/SeamlessIntegrationAnotherButton.vue

```diff
@@ -10,7 +10,7 @@
   border: none;
   color: #ffffff;
   transition-duration: 0.4s;
-  overflow: hidden;
+  overflow: hidden !important;
```

Comment:
Why important here? It doesn't matter - I'm just curious.

Reply:
fix animation effects for seamless btn